DragonFruit.Six.Api/Extensions/GeneralStatsExtentions.cs

```diff
@@ -7,35 +7,43 @@
 using DragonFruit.Six.Api.Entities;
 using DragonFruit.Six.Api.Deserializers;
 using DragonFruit.Six.Api.Requests;
+using DragonFruit.Six.Api.Utils;
 using Newtonsoft.Json.Linq;
 
 namespace DragonFruit.Six.Api.Extensions
 {
-    public static class StatsExtentions
+    public static class GeneralStatsExtentions
```

Comment:
Only just noticed this... Extentions instead of Extensions. Was already an issue before by the loops of it

Reply:
resolved in 76e85a53